addons/website_slides/views/slide_channel_tag_views.xml

```diff
@@ -27,19 +27,6 @@
         </field>
     </record>
 
-    <record id="slide_channel_tag_view_tree" model="ir.ui.view">
```

Comment:
Is there any reason removing the tree view ? Even if there is no specific action for tags anymore keeping a specific tree view is useful, if people craft custom menus or actions for example.

Reply:
I removed it since it wasn't being used anywhere else (i.e. to have cleaner code), but I understand your point. I added it back in.